src/util.js

```diff
@@ -66,3 +66,60 @@ exports.handleServerError = (ctx, operationFailureMsg, error, logger) => {
   ctx.body = {error: err}
   logger.error(err)
 }
+
+// Strip away opening and closing forward slash if they are present in path
+const removeEnclosingSlashes = path => {
+  path = path.trim()
+
+  if (path[0] === '/') {
+    path = path.substring(1)
+  }
+  if (path[path.length - 1] === '/') {
+    path = path.substring(0, path.length - 1)
+  }
+
+  return path
+}
+
+exports.extractRegexFromPattern = pattern => {
+  if (pattern[0] === '/') {
+    pattern = pattern.substring(1)
+  }
```

Comment:
it will not work there. It removes the trailing slash, which if present is added to the regex. Paths with trailing slashes will only match on patterns which expect trailing slashes

Reply:
it will not work here. It removes the trailing slash, which if present should be added to the regex as paths with trailing slashes will only match on patterns that expect trailing slashes